css/custom.css

```diff
@@ -153,3 +153,12 @@ a, a:hover {
     max-width: 1280px;
     padding: 20px 30px;
 }
+
+/* Snap h2 and h3 links below the header */
+h2 {
+  scroll-margin-top: 62px;
```

Comment:
Can we do 82px instead? This gives 20px between the header and the text, which is the same amount of space at the top of the content. Should probably include a comment about the significance of 62px as well.

Reply:
updated and added